engine/render/src/render/font_renderer_default.cpp

```diff
@@ -314,10 +316,12 @@ namespace dmRender
 
                 if (glyph->m_Width > 0)
                 {
-                    float f_width        = glyph->m_Width;
+                    float f_width        = glyph->m_ImageWidth;
                     float f_descent      = glyph->m_Descent;
                     float f_ascent       = glyph->m_Ascent;
                     float f_left_bearing = glyph->m_LeftBearing;
+                    // This is the difference in size of the glyph image and glyph size
+                    float f_size_diff    = glyph->m_ImageWidth - glyph->m_Width;
```

Comment:
The glyph bitmap may be larger than the glyph width. This assumes the glyph is positioned in the center of the bitmap.